dev/TreeView/TestUI/TreeViewPage.xaml.cs

```diff
@@ -243,18 +243,48 @@ private void GetRootNodeChildrenOrder(TreeView tree)
                 sb.Append(GetNodeContent(currentNode) + " | ");
             }
 
-            Results.Text = sb.ToString();
+            return sb.ToString();
+        }
+
+        private String GetItemsSourceOrder()
+        {
+            StringBuilder sb = new StringBuilder();
+            Stack<TreeViewItemSource> pendingItems = new Stack<TreeViewItemSource>();
+            pendingItems.Push(TestTreeViewItemsSource[0]);
+            while (pendingItems.Count > 0)
+            {
+                var currentItem = pendingItems.Pop();
+                var children = currentItem.Children;
+                var size = children.Count;
+                for (int i = 0; i < size; i++)
+                {
+                    pendingItems.Push(currentItem.Children[size - 1 - i]);
```

Comment:
You declared a "children" variable but didn't need it here. Either use it or remove it.